src/TwigComponent/src/ComponentAttributes.php

```diff
@@ -64,6 +64,8 @@ function (string $carry, string $key) {
                     $value = 'true';
                 }
 
+                $key = str_replace('::', ':', $key);
```

Comment:
In a regex to enforce it's only for nested attributes ? (so after a word / before another one) ?